modules/gentemplates/common/.gitignore

```diff
@@ -0,0 +1 @@
+src/main/xtend-gen
```

Comment:
@tedepstein This should be added in the root .gitignore file instead and it can be only `xtend-gen` not `src/main/xtend-gen` like that it also includes `src/test/xtend-gen`. But that's odd that it happens because `xtend-gen` folder should be inside `target` folder.